### ISSUE TITLE
Various verbose output to clean

### ISSUE THREAD

**Issue by fradav** [diyabc/diyabc#29]
"sortie de do_snp"

**Comment by fradav** [comment]
https://github.com/diyabc/diyabc/releases/tag/3.0.5-beta1

**Comment by fradav** [comment]
Fixed in c38939d
Release https://github.com/diyabc/diyabc/releases/tag/v1.0-RF

**Comment by gdurif** [comment]
When simulating training set with no conditions or baddly set conditions, the program fill stdout (or stderr) with very long and inopportune output,  e.g.

```
locus 257    ordre[0][1] = 7
locus 257    ordre[0][0] = 17
locus 257    ordre[0][0] = 10
```

The log file can reach millions of lines and 10+ MB size, which totally froze the user interface.

**Comment by fradav** [comment]
Fixed in d0de6869bb203b495bc62ac4f5e3ca06d5a0ac4b
